frontends/relay/dahlia_impl.py

```diff
@@ -441,6 +441,9 @@ def emit_components(func_defs: List[DahliaFuncDef]) -> str:
     created from the list of Dahlia function definitions.
     This does not include the import statement.
     """
+    if len(func_defs) == 0:
```

Comment:
I think `if not func_defs` is equivalent.

Reply:
Fixed. Thanks!